packages/lesswrong/components/ea-forum/UrlHintText.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+// We're not using Link, just useHistory
+// eslint-disable-next-line no-restricted-imports
+import { useHistory } from 'react-router-dom';
+import { Link } from '../../lib/reactRouterWrapper';
+import { registerComponent } from '../../lib/vulcan-lib';
+
+const linkPostPostPath = '/posts/8yDsenRQhNF4HEDwu/link-posting-is-an-act-of-community-service'
+
+/** Need a whole damn component to insert a link */
+const UrlHintText = () => {
+  const history = useHistory();
+  
+  return <>
+    Please write what you liked about the post, and consider sharing some relevant excerpts. If you have permission from the author, you can also copy in the entire post text. If you know the author's username you can add them as a co-author of this post in the "Options" menu below. You can find more guidelines{' '}
+    <Link
+      to={linkPostPostPath}
+      // This link gets removed as soon as focus shifts from the input,
+      // mousedown shifts focus from the input, and prevents the link from
+      // being clicked, so we immediately click the link on mouseDown.
+      onMouseDown={e => {
+        if (e.ctrlKey || e.metaKey) {
```

Comment:
This is potentially a bit weird - on a Mac I'd expect ctrl+click to open a context menu, but here it opens the link in a new tab. OTOH, this is an extremely minor issue which probably nobody will ever notice.

Reply:
Yeah, I copied this from what we did in the Aletheia codebase back in the day. I agree it's not ideal, but not worth figuring out what the right solution is.